### PR TITLE
Adjust design based on Design Review

### PR DESCRIPTION
**Pull request by Chartman123**
- Change Icons for results and links based on a decision in a design review
- Switch public and internal links in Sidebar
- Left align questions and header in create view

Signed-off-by: Christian Hartmann [chris-hartmann@gmx.de](mailto:chris-hartmann@gmx.de)

### CODE REVIEW

File: src/components/Questions/Question.vue [hunk 1]
```diff
@@ -260,7 +260,7 @@ export default {
 	&__drag-handle {
 		position: absolute;
 		display: flex;
-		left: 0;
+		left: -24px;
```
Comment (Chartman123) [inline src/components/Questions/Question.vue:263]: This moves the drag handle to an invisible area on mobile devices / small screens... Better solutions?

File: src/components/SidebarTabs/SharingSidebarTab.vue [hunk 2]
```diff
@@ -27,29 +27,10 @@
...
-					{{ t('forms', 'Copy to clipboard') }}
-				</NcActionButton>
-			</NcActions>
-		</div>
-
 		<!-- Public Link -->
 		<div v-if="!hasPublicLink && appConfig.allowPublicLink" class="share-div share-div--link">
 			<div class="share-div__avatar">
-				<IconLink :size="22" />
+				<IconLinkVariant :size="22" />
```
Comment (jotoeri) [inline src/components/SidebarTabs/SharingSidebarTab.vue:33]: I'd say 22 is too big with the new icon. Lets get down to 20 ;)

File: src/views/Create.vue [hunk 3]
```diff
@@ -412,6 +412,7 @@ export default {
 		flex-direction: column;
 		margin-top: 44px;
 		margin-bottom: 24px;
+		margin-left: 60px;
```
Comment (jotoeri) [inline src/views/Create.vue:415]: Should be 56px from my feeling of alignment? :wink:

Comment (Chartman123) [reply src/views/Create.vue:415]: That was my first value, but the alignment lines in FF for the questions section went directly through the letters in the header in this case...

Comment (jotoeri) [reply src/views/Create.vue:415]: Uhm, i see.
But that is only the title. For the alignment wrt. to the Form content (i.e. description and Infotext), the 56px match. And that the title is slightly offset should be fine or even intentional. Then the content is aligned and subordinated to the title.

Comment (Chartman123) [reply src/views/Create.vue:415]: Ok :) Than I'll change it to the 56px again 👍🏻

File: src/components/SidebarTabs/SharingSidebarTab.vue (hunk 2 above)
Comment (Chartman123) [reply src/components/SidebarTabs/SharingSidebarTab.vue]: Fixed 👍🏻

File: src/views/Create.vue (hunk 3 above)
Comment (Chartman123) [reply src/views/Create.vue]: Fixed 👍🏻

**Review by jotoeri — APPROVED** [review]

**Pull request merged**